### PR TITLE
Fix decorator usage in tests 

### PR DESCRIPTION
**Pull request by niboshi**

### CODE REVIEW

File: cupy/testing/helper.py [hunk 1]
```diff
@@ -137,6 +137,10 @@ def _contains_signed_and_unsigned(kw):
 
 def _make_decorator(check_func, name, type_check, accept_error, sp_name=None,
```
Comment (emcastillo) [inline cupy/testing/helper.py]: tests use `accept_error` as kwarg, should we force the arguments to be kwarg only?

Comment (niboshi) [reply cupy/testing/helper.py]: `_make_decorator` is an internal function, and there are only a few callers. I don't think there's any reason to make it keyword-only.

Comment (niboshi) [reply cupy/testing/helper.py]: If you think that increases readability, I won't object that, though.

Comment (emcastillo) [reply cupy/testing/helper.py]: with the reduced number of cases and the unlikely addition of more, it is ok to leave it as-is. I was suggesting it only to enforce using the same style in the code base, but as you said probably it won't be used in other places.

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by niboshi** [comment]
Fixed test failures.
PTAL

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by niboshi** [comment]
@emcastillo Backport to v6?

**Comment by emcastillo** [comment]
I am trying it, but the backport script breaks when pushing the changes ...
I am looking into it now

**Pull request merged**